flexmeasures/data/queries/utils.py

```diff
@@ -37,6 +41,43 @@ def create_beliefs_query(
     return query
 
 
+def potentially_limit_query_to_account_assets(
+    query: Query, account_id: Optional[int]
+) -> Query:
+    """
+    Filter out any assets that are not in this account, or not in the current
+    user's account (for non-admins or non-CLI users).
```

Comment:
Too complicated sentence. Could this be better?
```
"""Filter out all assets that are not in the current user's account.
For admins and CLI users, no assets are filtered out, unless an account_id is set.

:param account_id: if set, all assets that are not in the given account will be filtered out (only works for admins and CLI users)
"""
```

Reply:
I agree.